### PR TITLE
Remotes/origin/tf skeleton

### PR DESCRIPTION
**Pull request by gds-ahine**

### CODE REVIEW

**Comment by gds-ahine** [comment]
They shouldn't, I can see that I've moved .tfvars from .gitignore, I can't imagine why I done that. Will fix and update. Thanks

File: terraform/deployable/health_monitor_lambda.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    
+def notify_slack_sns(slack_sns_arn, slack_sns_message):
+	""" Send message to Slack SNS """
+	send_to_sns(slack_sns_arn, slack_sns_message)
+
+    
+def notify_dashboard_sns(dashboard_sns_arn, dashboard_sns_message):
+	""" Send message to Dashboard SNS """
+	send_to_sns(dashboard_sns_arn, dashboard_sns_message)
+
```
Comment (danjoneslf) [inline terraform/deployable/health_monitor_lambda.py]: Are these functions necessary - are you thinking we might need to do something inside these in future? 

We could pull the topic ARN env var inside each respective function so you only had to pass the message or we could just invoke send_to_sns directly.

File: terraform/deployable/health_monitor_lambda.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			"AlarmName": message['AlarmName'],
+			"StateChangeTime": message['StateChangeTime'],
+			"OldStateValue": message['OldStateValue'],
+			"NewStateValue": message['NewStateValue'],
+			"NewStateReason": message['NewStateReason'],
+			"Region": message['Region'],
+			"AlarmDescription": message['AlarmDescription'],
+			"MetricName": message['Trigger']['MetricName'],
+			"NameSpace": message['Trigger']['Namespace']
+		}
```
Comment (danjoneslf) [inline terraform/deployable/health_monitor_lambda.py]: Something's gone wrong with the spacing here: pylint won't like it.

File: terraform/deployable/health_monitor_lambda.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+""" Health monitoring lambda """
+import json
+import logging
+import os
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```
Comment (danjoneslf) [inline terraform/deployable/health_monitor_lambda.py]: I wonder if we should pass the log level in from the env so we can have different log level between test and production.

**Pull request closed without merging**